.Jules/palette.md

```
@@ -0,0 +1,3 @@
+## 2024-05-16 - Add missing aria-labels to utility buttons
```

markdownlint-cli2 (0.21.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)